src/app/globals.css

```
@@ -156,6 +157,45 @@ body {
   color: transparent;
 }
 
+/* Hero display: transparent fill with a theme-aware stroke */
+@utility text-stroke-name {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)